luigi_pipeline/seqr_loading.py

```diff
@@ -11,12 +11,37 @@
 from lib.model.seqr_mt_schema import SeqrVariantSchema, SeqrGenotypesSchema, SeqrVariantsAndGenotypesSchema
 
 logger = logging.getLogger(__name__)
+STANDARD_CONTIGS = {'1','10','11','12','13','14','15','16','17','18','19','2','20','21','22','3','4','5','6','7','8','9','X','Y'}
 
 
 def check_if_path_exists(path, label=""):
     if (path.startswith("gs://") and not hl.hadoop_exists(path)) or (not path.startswith("gs://") and not os.path.exists(path)):
         raise ValueError(f"{label} path not found: {path}")
 
+def contig_check(mt, threshold):
+    row_dict = mt.aggregate_rows(hl.agg.counter(mt.locus.contig))
+    contigs_set = set(row_dict.keys())
+    missing_contigs = STANDARD_CONTIGS - contigs_set
+    invalid_contigs = {contig: None for contig in missing_contigs}
+    result = True
+    
+    for k,v in row_dict.items():
+        if k not in STANDARD_CONTIGS:
+            invalid_contigs[k] = ''
+        if v < threshold:
```

Comment:
Yeah the original idea was to start with a basic check, especially since so far our issues have been all-or-nothing. I think in the future we can update this for chromosome-specific threshholds